### PR TITLE
Added new 42 keymap for the cantor

### PR DESCRIPTION
**Pull request by csolje**
## Description
Added the default qwerty layout and changed the layer switching to always be on and
added Mod-Tap for the space and enter keys.

## Types of Changes
New keymap

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cantor/config.h [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 #include "config_common.h"
 
+/* Tapping Term */
+#define TAPPING_TERM 175
+
```
Comment (zvecr) [inline keyboards/cantor/config.h:10]: Needs to be in a keymap level `config.h`
```suggestion
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -26,5 +26,6 @@
     "python.formatting.provider": "yapf",
     "[json]": {
 	    "editor.formatOnSave": false
-    }
+    },
+    "cmake.sourceDirectory": "${workspaceFolder}/lib/googletest"
```
Comment (zvecr) [inline .vscode/settings.json:30]:
```suggestion
    }
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/cantor/keymaps/zeromist/keymap.c [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    MPASTE,
+    MLCKSCN,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case MCOPY:
+        if (record->event.pressed) {
+            // Macro for copy
+            SEND_STRING(SS_LCTL("c"));
```
Comment (drashna) [inline keyboards/cantor/keymaps/zeromist/keymap.c:17]: you can use `C(KC_C)` in yur keymap, instead of this.

Comment (csolje) [reply keyboards/cantor/keymaps/zeromist/keymap.c:17]: So this one would look like this SEND_STRINGS(C(KC_C)) ?

Comment (drashna) [reply keyboards/cantor/keymaps/zeromist/keymap.c:17]: No.  I mean, you can put `C(KC_C)` directly into your keymap.  No need for a macro at all.

File: keyboards/cantor/keymaps/zeromist/keymap.c [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+      *                                │GUI├───┐           ┌─────┤Alt│
+      *                                └───┤Bsp├───┐   ┌───┤TO(0)├───┘
+      *                                    └───┤ENT│   │SPC├─────┘
+      *                                        └───┘   └───┘
+      */
+    [2] = LAYOUT_split_3x6_3(
+      //--------------------------------------------------------------         ---------------------------------------------------------------------------------
+        KC_MSTP, KC_MPLY,    KC_MPRV,   KC_MNXT,  KC_NO,    KC_NO,              KC_PGDOWN,    KC_PGUP,    KC_INS,      KC_PSCR,    KC_NO,    KC_BSPC,
+      //--------------------------------------------------------------         ---------------------------------------------------------------------------------
+        KC_NO,   KC_HOME,    KC_END,    MCOPY,    MPASTE,   KC_CAPS,            KC_LEFT,    KC_DOWN,   KC_UP,    KC_RGHT,    KC_DEL,   KC_NO,
```
Comment (drashna) [inline keyboards/cantor/keymaps/zeromist/keymap.c:107]:
```suggestion
        KC_NO,   KC_HOME,    KC_END,    C(KC_C),    C(KC_V),   KC_CAPS,            KC_LEFT,    KC_DOWN,   KC_UP,    KC_RGHT,    KC_DEL,   KC_NO,
```

Comment (csolje) [reply keyboards/cantor/keymaps/zeromist/keymap.c:107]: sweet, that makes it alot easier

Comment (drashna) [reply keyboards/cantor/keymaps/zeromist/keymap.c]: Exactly! :)

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/cantor/keymaps/zeromist/keymap.c [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+// SPDX-License-Identifier: GPL-2.0
+
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    MCOPY = SAFE_RANGE,
+    MPASTE,
+    MLCKSCN,
+};
+
```
Comment (fauxpark) [inline keyboards/cantor/keymaps/zeromist/keymap.c:11]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**